products/compute/api.yaml

```diff
@@ -844,6 +847,7 @@ objects:
   - !ruby/object:Api::Resource
     name: 'ForwardingRule'
     kind: 'compute#forwardingRule'
+    min_version: alpha
```

Comment:
This should be moved down to the allPorts field, since ForwardingRules themselves are available in versions other than alpha

Reply:
Fixed.